tools/GSDumpGUI/Forms/frmMain.cs

```diff
@@ -107,15 +116,61 @@ public GSDumpGUI()
 
             _availableGsDumps.OnIndexUpdatedEvent += UpdatePreviewImage;
 
-            txtGSDXDirectory.DataBindings.Add(nameof(TextBox.Text), Settings, nameof(Settings.GSDXDir));
```

Comment:
It was necessary to remove model binding? Typically you want to separate view and model logic because on data level you don’t care how and where data is displayed and on view level you don’t care who is when changing data but just that you need to update views if data changes.

Reply:
I removed them because something was bringing the old path back after it was changed. Never happened again after I removed that. I could try adding it again and debug elsewhere.

Reply:
Yes model binding sucks heavily in winforms. At least for this framework version. Just keep it removed. I wanted to relaunch the app in an MVC way anyways at one magical day in future.